src/Symfony/Component/HttpFoundation/Session/Storage/Handler/PdoSessionHandler.php

```diff
@@ -775,13 +771,9 @@ private function getSelectSql(): string
     /**
      * Returns an insert statement supported by the database for writing session data.
      *
-     * @param string $sessionId   Session ID
-     * @param string $sessionData Encoded session data
-     * @param int    $maxlifetime session.gc_maxlifetime
```

Comment:
quite a lot of useful phpdoc descriptions have been removed :/

Reply:
Which of those would you like to keep?

Reply:
maxlifetime